autoload/matchup/matchparen.vim

```diff
@@ -670,8 +670,11 @@ function! s:do_offscreen_popup_nvim(offscreen) " {{{1
           \ 'height': &previewheight,
           \ 'focusable': v:false,
           \}
-    if get(g:matchup_matchparen_offscreen, 'border', 0)
-      let l:win_cfg.border = ['', '═' ,'╗', '║', '╝', '═', '', '']
+    let l:border = get(g:matchup_matchparen_offscreen, 'border', 0)
+    if l:border != v:null
+      let l:win_cfg.border = has('nvim-0.5')
+            \ && type(l:border) == v:t_string ?
```

Comment:
very minor, but I find it a bit easier to read to put the `?` operator on the next line

Reply:
Changed!